concepts/Common/DevExtreme CLI/20 DevExtreme Application/10 Create a New Application.md

```diff
@@ -30,14 +30,18 @@ Create a new <a href="https://devexpress.github.io/devextreme-react-template" ta
 Arguments:
 
 * `--layout`        
-Specifies the DevExtreme layout to add. Available values:
+Specifies the DevExtreme layout. Available values:
     * `side-nav-outer-toolbar` (default)
     * `side-nav-inner-toolbar`
 
-    Both layouts are illustrated in the [Layouts](/concepts/40%20Angular%20Components/30%20Application%20Template/10%20Layouts.md '/Documentation/Guide/Angular_Components/Application_Template/#Layouts') article.
+    Use the following topic to review both layouts: [Layouts](/concepts/40%20Angular%20Components/30%20Application%20Template/10%20Layouts.md '/Documentation/Guide/Angular_Components/Application_Template/#Layouts').
 
 * `--empty`     
 Specifies whether to skip sample views generation (default: `false`).
 
+* `--template` (React only)
+Specifies whether to create an application template with TypeScript support. Available values: `javascript` (default) and `typescript`.
```

Comment:
Can we make `javascript` (default) and `typescript` a bulleted list? For example:

```suggestion
Specifies whether to create an application template with TypeScript support. Available values:
* `javascript` (default)
* `typescript`
```